packages/node_modules/@webex/plugin-meetings/src/metrics/index.js

```diff
@@ -157,6 +169,36 @@ class Metrics {
     return payload;
   }
 
+  initMediaPayload(eventType, identifiers, options) {
+    const payload = {
+      eventId: uuid.v4(),
+      version: 1,
+      origin: {
+        buildType: 'prod',
+        name: 'endpoint',
+        networkType: 'unknown',
+        userAgent: this.getUserAgentInfo(),
+        clientInfo: {
+          clientType: options.clientType,
+          os: 'linux', // TODO: BUG AND DOESNT ACCEPT OSNAME ON CA -> bowser.osname || 'null'
+          osVersion: bowser.osVersion || '-1'
+        }
+      },
+      originTime: {
+        triggered: new Date().toISOString()
+      },
+      event: {
+        name: eventType,
+        canProceed: true,
+        identifiers,
+        intervals: [options.intervalData],
+        eventData: {webClientDomain: window.location.hostname}
```

Comment:
Do external users still want to use `webClientDomain`?

Reply:
Yeah, it's how we differentiate it on our dashboard, we don't want to change the key